python_modules/dagster-graphql/dagster_graphql/schema/backfill.py

```diff
@@ -532,6 +537,49 @@ def resolve_title(self, _graphene_info: ResolveInfo) -> Optional[str]:
     def resolve_description(self, _graphene_info: ResolveInfo) -> Optional[str]:
         return self._backfill_job.description
 
+    def resolve_logEvents(self, graphene_info: ResolveInfo, cursor: Optional[str] = None):
+        from ..schema.instigation import (
+            GrapheneInstigationEvent,
+            GrapheneInstigationEventConnection,
+        )
+        from ..schema.logs.log_level import GrapheneLogLevel
+
+        backfill_id = self._backfill_job.backfill_id
+        # TODO find a better way to keep the log key prefix in sync. maybe store as part of the asset backfill data?
+        backfill_log_key_prefix = ["backfill", backfill_id]
+
+        instance = graphene_info.context.instance
+
+        if not isinstance(instance.compute_log_manager, CapturedLogManager):
+            return GrapheneInstigationEventConnection(events=[], cursor="", hasMore=False)
+
+        if not instance.backfill_log_storage_enabled():
+            # maybe need to return something else to indicate that a setting needs to be changed
```

Comment:
resolve this TODO - I think enabling this for all dagster-implemented compute log managers won't be difficult, so maybe we can just turn it on by default